DuggaSys/accessed.php

```diff
@@ -135,22 +135,22 @@
 					<input type='hidden' id='uid' value='Toddler' />
 					<div class='flexwrapper'>
 						<div class="tooltipDugga"><span id="tooltipFirst" style="display: none;" class="tooltipDuggatext">  </span></div>
-						<input placeholder="Greger" class='textinput' type='text' id='addFirstname' onchange="tooltipFirst()" onkeyup="tooltipFirst()"/>
+						<input placeholder="Greger" class='textinput' type='text' id='placeholderID4' onchange="tooltipFirst()" onkeyup="tooltipFirst()"/>
```

Comment:
Bad ID, could be changed to something like 'addFirstname1' or something along these lines, to keep relevant id but not get duplicate ids.

Reply:
The whole form is a placeholder